source/_integrations/roborock.markdown

```diff
@@ -51,13 +51,14 @@ We are working on adding a lot of features to the core integration. We have reve
 
 ### How can I clean a specific room?
 We plan to make the process simpler in the future, but for now, it is a multi-step process.
-1) Enable debug logging for this integration and reload it.
-2) Search your logs for 'Got home data' and then find the attribute rooms.
-3) Write the rooms down; they have a name and 6 digit ID.
-4) Go to **Developer Tools** > **Services** > **Vacuum: Send Command**. Select your vacuum as the entity and 'get_room_mapping' as the command.
-5) Go back to your logs and look at the response to `get_room_mapping`. This is a list of the 6-digit IDs you saw earlier to 2-digit IDs. In your original list of room names and 6-digit IDs, replace the 6-digit ID with its pairing 2-digit ID.
-6) Now, you have the 2-digit ID that your vacuum uses to describe a room.
-7) Go back to **Developer Tools** > **Services** > **Vacuum: Send Command** then type `app_segment_clean` as your command and 'segments' with a list of the 2-digit IDs you want to clean. Then, add  `repeat` with a number (ranging from 1 to 3) to determine how many times you want to clean these areas.
+1) Make sure to first name the rooms in the Android app, otherwise they won't show up in the debug log.
+2) Enable debug logging for this integration and reload it.
```

Comment:
I understand this is needed right now, but it would be great if we didn't need this. :)

That said, we can improve on these instructions by telling how to enable debug logging.